### DISCUSSION TITLE
Is it possible to use RDS IAM authentication when using AWS Postgres RDS?

### DISCUSSION THREAD

**Discussion by gdamjan** [concourse/concourse#6808] [Ideas]
The process is explain here
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html
but the basic idea is that instead of configuring a static password for the RDS instance, and then copy/pasting it to concourse, the authentication is dynamic based on IAM roles applied to the EC2 instance.

The benefit is massively simplifying deployment when in AWS environment and using RDS, since you don't need to configure postgres password.

**Comment by taylorsilva** [comment]
This sounds super handy! We don't use AWS ourselves, so a bit hard for us to build and test something like this. We're generally focused on non-iaas specific things. Seems like a great thing to PR!
